ui/widgets/nuxeo-document-suggestion.js

```diff
@@ -305,7 +305,18 @@ import { escapeHTML } from './nuxeo-selectivity.js';
       if (typeof doc === 'string') {
         return `<span>${escapeHTML(doc)}</span>`;
       }
-      return `<a href="${this.urlFor('browse', doc.path)}">${escapeHTML(doc.title)}</a>`;
+      const routeKey =
+        (Nuxeo &&
+          Nuxeo.UI &&
+          Nuxeo.UI.config &&
+          Nuxeo.UI.config.router &&
+          Nuxeo.UI.config.router.key &&
+          Nuxeo.UI.config.router.key.document) ||
+        'path';
+      // XXX we keep `routeKey === 'path' && this.router.browse` to keep compat routers that override the document
```

Comment:
didn't we just add this compat in the behavior itself ?  was expecting to see just `this.urlFor(doc)` here

Reply:
You are right, this is a duplicate check. Will remove. Thanks.